frontend/next.config.js

```diff
@@ -1,9 +1,11 @@
 // Dependencies
-import path from 'path'
-import FilterWarningsPlugin from 'webpack-filter-warnings-plugin'
-import Dotenv from 'dotenv-webpack'
+const path = require('path')
+const FilterWarningsPlugin = require('webpack-filter-warnings-plugin')
+const Dotenv = require('dotenv-webpack')
 
-export default {
+const { PHASE_DEVELOPMENT_SERVER } = require('next/constants')
```

Comment:
Can you add some comments to describe what are you doing? Remember that I need to do a video of all these new changes, but I want to make sure I understand what's going on :)